intel_extension_for_transformers/transformers/runtime/test/pytest/requirements.txt

```diff
@@ -1,11 +1,11 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
 accelerate
 datasets
-intel-extension-for-pytorch==2.0.0
+intel-extension-for-pytorch==2.3.0
```

Comment:
1. why tests are inside the itrex package @zhentaoyu 
2. why bind torch version here? different parts of ITREX depends on different version of torch @kevinintel